src/gp_doctest.cpp

```diff
@@ -1562,3 +1562,31 @@ TEST_CASE("NNI Engine: NNI Likelihoods") {
     nni_count++;
   }
 }
+
+// Builds a SubsplitDAG. Uses a naive method that picks the first listed neighbor for
```

Comment:
I was confused for a while upon reading this explanation-- I thought that the first sentence implied that the main point of this test was building a subsplit DAG. 

We don't have to describe that detail... just have the first sentence tell us what the overall goal of the test is.

Reply:
Reworded test comment.